### DISCUSSION TITLE
Nested modal

### DISCUSSION THREAD

**Discussion by jimmyps** [vikramlearning/blazorbootstrap#228] [Q&A]
Is nested modal supported?

**Comment by gvreddy04** [comment]
@jimmyps Nested modals are not supported. Please share more information about your requirement so that I can suggest alternate options. If it works, you can try using an `offcanvas` and `modal` combination.

An offcanvas is a panel that can be opened from the side of the screen. A modal is a full-screen window that covers the entire content area. By using an offcanvas and modal combination, you can provide users with a more flexible and user-friendly experience.

Offcanvas: https://demos.blazorbootstrap.com/offcanvas
Modals: https://demos.blazorbootstrap.com/modals
